cfbs/commands.py

```diff
@@ -1067,9 +1068,21 @@ def _compare_list(a, b):
             return 1
 
     path = os.path.join(name, "input.json")
-    log.debug("Writing json to file '%s'" % path)
-    write_json(path, data)
 
+    log.debug("Comparing with data already in file '%s'" % path)
+    old_data = read_json(path)
+    if old_data == data:
+        log.debug("Input data for '%s' unchanged, nothing to write / commit" % name)
+        return 0
```

Comment:
This should, ideally, be done before the more expensive checks/comparison above.

Reply:
I was thinking it makes sense to always do the expensive checks (validation), even if the (potentially bad) data is already there.